flatpak/com.github.qarmin.Czkawka.yaml

```diff
@@ -0,0 +1,36 @@
+app-id: com.github.qarmin.Czkawka
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: czkawka_gui
+rename-icon: com.github.qarmin.czkawka
+rename-desktop-file: com.github.qarmin.czkawka.desktop
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--filesystem=host"
+- "--device=dri"
+build-options:
+  append-path: "/usr/lib/sdk/rust-stable/bin"
+  env:
+    CARGO_HOME: "/run/build/czkawka_gui/cargo"
+modules:
+- name: czkawka_gui
+  buildsystem: simple
+  build-commands:
+  - cargo --offline fetch --manifest-path Cargo.toml
+  - cargo --offline build --release
+  - install -Dm755 ./target/release/czkawka_gui -t /app/bin/
+  - install -Dm644 ./data/icons/com.github.qarmin.czkawka.svg -t /app/share/icons/hicolor/scalable/apps/
+  - install -Dm644 ./pkgs/com.github.qarmin.czkawka.desktop -t /app/share/applications/
+  - install -Dm644 ./flatpak/com.github.qarmin.Czkawka.metainfo.xml -t /app/share/metainfo
```

Comment:
I have error 
```
Running: install -Dm644 ./flatpak/com.github.qarmin.Czkawka.metainfo.xml -t /app/share/metainfo
install: cannot stat './flatpak/com.github.qarmin.Czkawka.metainfo.xml': No such file or directory
```

Reply:
https://github.com/qarmin/czkawka/pull/203#issuecomment-761699805. It cannot work until it's merged. You can change the URL to my Czkawka fork for testing.

Reply:
Oh, yeah, I forgot about it